{
  loadText("fun foo() { throw Exception(); }");
  Method main=generateFunction();
  assertThrows(main,Exception.class,null);
}
